client/src/components/Inputs/TextInput.tsx

```diff
@@ -0,0 +1,25 @@
+type TextInputProps = {
+  type: string;
+  name: string;
+  id: string;
+  placeholder: string;
+};
+
+const TextInput = ({
```

Comment:
```suggestion
const TextInput: React.FC<TextInputProps> = ({
```
Giống với button, input cũng có otherProps nha
Như autocomplete, value,...